README.md

```diff
@@ -35,13 +35,16 @@ This project uses `semver`.
 ## Usage
 
 ```js
-new Fingerprint2().get(function(result, components) {
-  console.log(result) // a hash, representing your device fingerprint
-  console.log(components) // an array of FP components
-})
+setTimeout(function () {
```

Comment:
Would requestIdleCallback be as good as setTimeout here? Should we recommend it instead?

Reply:
requestIdleCallback has its uses, but for now would not recommend it over setTimeout, as it is still a draft

Reply:
Why use it in the demo HTML then?

Reply:
requestIdleCallback executes as soon as the browser is idle without having to guess a good timing with setTimeout. It also executes after all the other setTimeouts without having to know their timings because it has lower priority.

It basically is a "execute as soon as possible, after everything else has loaded and executed".

But the downside is that it is less predictable. For example if you make a request that should contain the fingerprint in a setTimeout of 5 seconds the IdleCallback may have executed, however if fingerprint was in a setTimeout of 500ms you have correct order guarantee.

To get best of both worlds, one could use an [idle-until-urgent helper](https://philipwalton.com/articles/idle-until-urgent/)

Reply:
Of course it depends how the application is coded.

The main purpose of our html file is to display the fingerprint and that's it.

Of course setTimeout is still used as a fallback.

Reply:
I just checked https://www.w3.org/TR/requestidlecallback/ and it not a draft anymore but a Proposed Recommendation.

So we could recommend it in the readme also, what do you think ?
 * requestIdleCallback **optimal** **may require application coordination** **only firefox and chrome**
 * setTimeout **stable** **not optimal**

Reply:
Yes, and add a comment to the demo HTML that people should go read the comment in the README.

Reply:
done